docs/DG_Diagrams/StorageComponent.puml

```diff
@@ -0,0 +1,9 @@
+@startuml
+
+Rectangle Storage {
+abstract class Storage
+Storage <|- FavStorage
+Storage <|-left FreqStorage
+}
+
```

Comment:
Since both attribute and method boxes are empty in the classes, I think it is possible to exclude the empty boxes, so just the name of the classes.
Also, remember to remove the icons as mentioned by the prof since it is not the standard notation.

These 2 suggestions apply for all the other diagrams as well.